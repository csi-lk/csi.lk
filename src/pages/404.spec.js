import React from 'react'

import NotFoundPage from './404'

describe('Page: Not Found (404)', () => {
  it('renders', () => {
    const wrapper = shallow(<NotFoundPage />)
    expect(wrapper).toMatchSnapshot()
  })
})
