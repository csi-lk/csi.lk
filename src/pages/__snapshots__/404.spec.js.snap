// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page: Not Found (404) renders 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NotFoundPage />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1>
          NOT FOUND
        </h1>,
        <p>
          You just hit a route that doesn't exist... the sadness.
        </p>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "NOT FOUND",
        },
        "ref": null,
        "rendered": "NOT FOUND",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "You just hit a route that doesn't exist... the sadness.",
        },
        "ref": null,
        "rendered": "You just hit a route that doesn't exist... the sadness.",
        "type": "p",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1>
            NOT FOUND
          </h1>,
          <p>
            You just hit a route that doesn't exist... the sadness.
          </p>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "NOT FOUND",
          },
          "ref": null,
          "rendered": "NOT FOUND",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "You just hit a route that doesn't exist... the sadness.",
          },
          "ref": null,
          "rendered": "You just hit a route that doesn't exist... the sadness.",
          "type": "p",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
